auth/index.html

```diff
@@ -1322,6 +1322,16 @@ <h2>Message log</h2>
         TKHQ.sendMessageUp("EMBEDDED_PUBLIC_KEY", targetPubHex, requestId);
       };
 
+      /**
+       * Removes the embedded key from local storage.
+       */
+      var onResetEmbeddedKey = async function (requestId) {
+        await TKHQ.resetEmbeddedKey();
+        document.getElementById("embedded-key").value = "";
+
+        TKHQ.sendMessageUp("EMBEDDED_KEY_RESET", null, requestId);
```

Comment:
Side note: these event names that we pass back up (e.g. `EMBEDDED_KEY_RESET`) aren't really used any more; instead we use `requestId` to match up + handle requests on the parent (iframe client) side